src/components/DailyForcast.vue

```diff
@@ -0,0 +1,44 @@
+<template>
+  <ul>
+    <li v-for="day in days" :key="day.dt">
+      <forcast-item
+        class="m-2"
+        :dateTime="dtToDate(day.dt)"
+        dateTimeFormat="E"
+        :imageSrc="iconToUrl(day.weather[0].icon)"
+        :high="day.temp.max"
+        :low="day.temp.min"
+      ></forcast-item>
+    </li>
+  </ul>
+</template>
+<script lang="ts">
+import { Component, Vue } from 'vue-property-decorator';
+import { Store } from 'vuex';
+import { RootState } from '@/store/types';
+import ForcastItem from '@/components/ForcastItem.vue';
+import openWeatherService from '@/services/openWeatherService';
+
+@Component({
+  components: { ForcastItem },
+  methods: {
+    dtToDate: openWeatherService.dtToDate,
+    iconToUrl: openWeatherService.iconToUrl,
+  },
+})
+export default class DailyForcast extends Vue {
```

Comment:
Forcast should be spelled For**e**cast with an E.... I had to google it to be sure

Reply:
Thanks for catching that!